release/models/qos/openconfig-qos-elements.yang

```diff
@@ -1231,6 +1237,18 @@ submodule openconfig-qos-elements {
             term - such that packets that arrive in the queue are
             immediately serviced.";
         }
+        enum DWRR {
+          description
+            "This scheduler term is considered as a deficit weighted
+            round robin term - such that packets that arrive in the queue are
+            serviced based on deficit counter with weighted round robin fashion.";
+        }
+        enum WRR {
```

Comment:
With the existing model, it seems like setting priority was optional, and at least for our mappings, I think that we defaulted to WRR if STRICT priority wasn't explicitly set.  Is the intention with this change that priority should always be set?  Would it make sense to mark WRR as the default so that this change is backwards compatible from an existing implementation perspective (with the underlying assumption that other implementations map similarly)?

Reply:
It seems like OpenConfig prefers not to define defaults, at least according to https://github.com/openconfig/public/pull/1165#discussion_r1813524909.